cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -197,7 +197,15 @@ def delete_org(self):
 
         if p.returncode:
             message = f"Failed to delete scratch org: \n{''.join(stdout)}"
-            raise ScratchOrgException(message)
+            message += "\nPerhaps it was already deleted?"
+            if force:
+                message += "\nRemoving org regardless."
+                self.logger.info(message)
```

Comment:
I think that suppressing the exception and showing this message belongs in `org_remove`, not here. It would be possible to call `org_config.delete_org(force=True)` in a context that does not proceed to remove the org.

Reply:
I'm confused. I intended the force=True as "remove the org even if you can't delete it". So why would you want to use it in a context where you did not want to remove the org?

Reply:
I agree with that -- which is why I don't think it makes sense to make it possible to use it without removing the org. Either the actual removal of the org from the keychain should also become the responsibility of this function (when `force` is true) or the log message about removing it should move to where it is removed.

Reply:
Also this method could be called from a web app and should not be emitting a message (below) that assumes it's running in the CLI context.